internal/mage/util/util.go

```diff
@@ -49,8 +53,8 @@ func RepositoryGoCodeOnly(c *dagger.Client) *dagger.Directory {
 			// modules
 			"**/go.mod",
 			"**/go.sum",
-			"**/go.work",
-			"**/go.work.sum",
+			// "**/go.work",
+			// "**/go.work.sum",
```

Comment:
Did you mean to delete these?

Reply:
Maybe. I'm not sure if anyone is using this, or if it's a leftover from when we were still figuring out how to bootstrap our CI stack. I had to remove these because my `go.work` points to `../foo` which doesn't get picked up and causes all the builds to fail.

For me the fastest feedback loop is doing dev/test with native `go test` etc., and then using `./hack/make test` as the final "about to push" run, where ignoring `go.work` and `go.work.sum` gives me a valuable reminder to actually bump the dependencies in `go.mod`.

cc @sipsma @marcosnils @jlongtine @helderco @aluzzardi to see if anyone's depending on this and whether I can remove it. :pray: